server/lib/sync-agent.js

```diff
@@ -260,8 +260,16 @@ export default class SyncAgent {
         }
       }
 
+      const omitTraits = ["external_id", "updated_at"];
```

Comment:
@KgaevskiT should we omit account_id in the traits list here ?

Reply:
Only if you are importing users : https://github.com/hull-ships/hull-sql/pull/43/files#diff-03b9ccd5863140a4d08c48c3699cc4c4R268